indico/modules/events/editing/client/js/components/FileManager/index.jsx

```diff
@@ -114,7 +114,20 @@ function FileType({uploadURL, fileType, uploads}) {
   const ref = useRef(null);
   return (
     <div styleName="file-type">
-      <h3>{fileType.name}</h3>
+      <h3>
+        {fileType.name}
+        <Popup
+          position="bottom center"
+          content={
+            fileType.filenameTemplate !== null ? (
+              <Translate>Filenames must have the format: {fileType.filenameTemplate}</Translate>
```

Comment:
You cannot interpolate values like this in translated strings; if you search for `<Param` in the codebase you'll find examples how to use parameters there. You can also use this to wrap the template in `<code>` (using the `wrapper` prop of `Param`)